openslides/core/migrations/0003_auto_20160803_1058.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-08-03 10:58
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import migrations
+from openslides.core.models import Projector
+
+
+def add_speaker_overlay(apps, schema_editor):
+    """
+    Adds a list of speaker overlay to the existing projector id=1
+    """
+    projector = Projector.objects.get(id=1)
+    soverlay = {}
+    soverlay_uuid = uuid.uuid4().hex
+    soverlay[soverlay_uuid] = {
+        'name': 'core/speakeroverlay',
+        'stable' : True,
+        'uuid' : soverlay_uuid}
+    projector.elements.update(soverlay)
```

Comment:
Try this:
```python
# -*- coding: utf-8 -*-
# Generated by Django 1.9.7 on 2016-08-03 10:58
from __future__ import unicode_literals

import uuid

from django.db import migrations, models


def add_speaker_overlay(apps, schema_editor):
    """
    Adds a list of speakers overlay to the existing projector id=1.
    """
    # We disconnect the inform_changed_data_receiver to disabled the
    # autoupdate mechanism during migrations.
    models.signals.post_save.disconnect(
        dispatch_uid='inform_changed_data_receiver')

    # We get the model from the versioned app registry;
    # if we directly import it, it will be the wrong version.
    Projector = apps.get_model('core', 'Projector')
    projector = Projector.objects.get(id=1)
    projector.config[uuid.uuid4().hex] = {
        'name': 'core/speakeroverlay',
        'stable' : True}
    projector.save()


class Migration(migrations.Migration):

    dependencies = [
        ('core', '0002_session'),
    ]

    operations = [
        migrations.RunPython(add_speaker_overlay)
    ]
```